src/renderer/vt/XtermEngine.cpp

```diff
@@ -105,7 +105,7 @@ XtermEngine::XtermEngine(_In_ wil::unique_hfile hPipe,
         // by prepending a cursor off.
         if (_lastCursorIsVisible)
         {
-            _buffer.insert(0, "\x1b[25l");
+            _buffer.insert(0, "\x1b[?25l");

```

Comment:
.. lol, how long has this been like this, and does it map to any bugs in our repo?

Reply:
and is there a constant val we can use somewhere

Reply:
It was mentioned in #3093, and I had a branch that fixed this before ([dev/migrie/b/be-better-at-hiding](https://github.com/microsoft/terminal/tree/dev/migrie/b/be-better-at-hiding)), but I have no memory of why that was never fixed